postmark/README.md

```diff
@@ -0,0 +1,72 @@
+# Postmark
+
+## Overview
+
+[Postmark][1] is an email delivery service for sending and tracking application emails. It offers a scalable alternative to traditional SMTP and provides essential metrics like sent, opened, bounced, and spam complaint rates. This enables efficient email management and monitoring.
+
+To gain insights into Postmark's broadcast and transactional message streams, integrate Postmark with Datadog using [webhooks][2].
+
+## Setup
+
+Follow the instructions below to configure this integration for your Postmark account.
+
+
+### Configuration
+
+#### Enable open and link tracking in server settings for the message streams
+Follow these steps to enable both features:
+
+1. Log in to your [Postmark account][3]. This redirects you to the [servers page][4].
+2. Select the desired server. This redirects you to the **Message Streams page**.
+3. In the navigation panel, click the **Settings** tab.
+4. In the Tracking section, enable both **Open tracking** and **Link tracking**.
+
+#### Webhook configuration steps for broadcast message streams
```

Comment:
can we combine the instructions for message streams and transcation message streams into the same steps and just differentiate where the instructions are different? Looking at the config instructions, both webhook config steps are basically identical.

Reply:
Done